src/renderer/components/cluster-manager/bottom-bar.tsx

```diff
@@ -63,7 +64,10 @@ export class BottomBar extends React.Component {
 
   render() {
     return (
-      <div className="BottomBar flex gaps">
+      <div className={styles.BottomBar}>
+        <div className={styles.item}>
+          <ActiveHotbarName/>
+        </div>
```

Comment:
I don't think we should special case this, why not register it? Futhermore, it probably would be a good idea for extensions to be able to choose which side they are on (defaulting to right)